### ISSUE TITLE
Wrong value for ingress "prefix" variable

### ISSUE THREAD

**Issue by mpangburn34** [criblio/helm-charts#200]
The default value in logstream-leader/values.yml for the `prefix` variable is `/*`. According to [Kubernetes docs](https://kubernetes.io/docs/concepts/services-networking/ingress/#path-types), the value `/` means all paths. When testing with the default value in my environment, nginx would give a 404 error on all paths. Once I changed the value to `/`, the reverse proxy worked as expected.

**Comment by bdalpe** [comment]
You are correct @mpangburn34. The prefix is implementation specific and may require a change to work correctly.

The default value of /* has been maintained for historical reasons and probably won't be changed in this chart as at some point the "logstream" name will be deprecated.
